recipe/build_opencv.bat

```diff
@@ -50,92 +51,95 @@ rem A number of data files are downloaded when building opencv contrib.
-    -DWITH_TENGINE=0                                                                ^
-    -DWITH_GSTREAMER=1                                                              ^
-    -DWITH_MATLAB=0                                                                 ^
-    -DWITH_TESSERACT=0                                                              ^
-    -DWITH_VA=0                                                                     ^
-    -DWITH_VA_INTEL=0                                                               ^
-    -DWITH_VTK=0                                                                    ^
-    -DWITH_GTK=0                                                                    ^
-    -DWITH_QT=%QT%                                                          ^
-    -DWITH_GPHOTO2=0                                                                ^
-    -DWITH_WIN32UI=0                                                                ^
-    -DINSTALL_C_EXAMPLES=0                                                          ^
-    -DOPENCV_EXTRA_MODULES_PATH=%UNIX_SRC_DIR%/opencv_contrib/modules               ^
-    -DPYTHON_EXECUTABLE=%UNIX_PREFIX%/python                                        ^
-    -DPYTHON_INCLUDE_PATH=%UNIX_PREFIX%/include                                     ^
-    -DPYTHON_PACKAGES_PATH=%UNIX_SP_DIR%                                            ^
-    -DPYTHON_LIBRARY=%UNIX_PREFIX%/libs/%PY_LIB%                                    ^
-    -DOPENCV_SKIP_PYTHON_LOADER=1                                                   ^
-    -DBUILD_opencv_python2=0                                                        ^
-    -DPYTHON2_EXECUTABLE=""                                                         ^
-    -DPYTHON2_INCLUDE_DIR=""                                                        ^
-    -DPYTHON2_NUMPY_INCLUDE_DIRS=""                                                 ^
-    -DPYTHON2_LIBRARY=""                                                            ^
-    -DPYTHON2_PACKAGES_PATH=""                                                      ^
-    -DOPENCV_PYTHON2_INSTALL_PATH=""                                                ^
-    -DBUILD_opencv_python3=1                                                        ^
-    -DPYTHON3_EXECUTABLE=%UNIX_PREFIX%/python                                       ^
-    -DPYTHON3_INCLUDE_PATH=%UNIX_PREFIX%/include                                    ^
-    -DPYTHON3_LIBRARY=%UNIX_PREFIX%/libs/%PY_LIB%                                   ^
-    -DPYTHON3_PACKAGES_PATH=%UNIX_SP_DIR%                                           ^
-    -DOPENCV_PYTHON3_INSTALL_PATH=%UNIX_SP_DIR%                                     ^
-    -DOPENCV_PYTHON_PIP_METADATA_INSTALL=ON                                         ^
-    -DOPENCV_PYTHON_PIP_METADATA_INSTALLER:STRING="conda"                           ^
-    ..
+
+cmake -LAH -B build -G "Ninja" -S .                                          ^
```

Comment:
`Actual changes` beyond whitespaces:

Added -DWITH_AVIF=1 (after -DCMAKE_CXX_STANDARD=17)
Changed -DWITH_FFMPEG=0 → -DWITH_FFMPEG=1
Changed -DWITH_GSTREAMER=1 → -DWITH_GSTREAMER=0
Added -DQT_SKIP_DEFAULT_TESTCASE_DIRS=0 (after -DWITH_QT=%QT%)
Added -DWITH_OBSENSOR=0
CMake invocation changed: First block uses old style cmake -G "Ninja" ... .. while second block uses -B build -G "Ninja" -S . (modern CMake syntax)

These are functional changes enabling/disabling OpenCV features (AVIF codec support, FFmpeg, GStreamer) and adding sensor support.

That means it's unnessesary to remove whitespaces because it's hard to read real changes.